### PR TITLE
kill jekyll; misc code cleanups

### PR DESCRIPTION
**Pull request by yjbanov**
/cc @devoncarew @sethladd 

Fixes https://github.com/flutter/flutter/issues/3714

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,24 +12,25 @@ To test this script locally:
...
-directory from this repo.
+The config file must be a JSON file under `dashboard_box` defining the following variables:
 
-The OAuth redirects are very specific.
+ * `ANDROID_DEVICE_ID` - the ID of the Android device used for performance testing
+ * `FIREBASE_FLUTTER_DASHBOARD_TOKEN` - authentication token to Firebase used to upload metrics (not needed for local testing)
+
+Example:
+
+```javascript
```
Comment (devoncarew) [inline README.md]: `json`?

File: bin/build.dart [hunk 2]
```diff
@@ -17,48 +18,48 @@ Future<Null> main(List<String> args) async {
 
   config = new Config(args.single);
 
-  Chain.capture(() {
-    build();
+  Chain.capture(() async {
+    section('Build started on ${new DateTime.now()}');
+    print(config);
+
+    if (!exists(config.dataDirectory)) {
```
Comment (devoncarew) [inline bin/build.dart]: no braces necessary

File: bin/build.dart [hunk 3]
```diff
@@ -17,48 +18,48 @@ Future<Null> main(List<String> args) async {
...
-      return;
-
-    File file = entity;
-    Map<String, dynamic> timelineSummary = JSON.decode(await file.readAsString());
-    summaries[path.basenameWithoutExtension(file.path)] = timelineSummary;
-  });
-  await config.summariesFile.writeAsString(JSON.encode(summaries));
+Future<Null> prepareDataDirectory() async {
+  // Backup old data
+  if (!exists(config.backupDirectory)) {
```
Comment (devoncarew) [inline bin/build.dart]: braces

Comment (devoncarew) [inline bin/build.dart]: here, and a few places below

File: lib/src/utils.dart [hunk 4]
```diff
@@ -18,9 +18,19 @@ void fail(String message) {
...
+    entity.deleteSync();
+}
+
+/// Remove recursively.
+void rrm(FileSystemEntity entity) {
+  if (entity.existsSync())
+    entity.deleteSync(recursive: true);
+}
+
+List<FileSystemEntity> ls(Directory directory) {
```
Comment (devoncarew) [inline lib/src/utils.dart]: method shorthand (`=>`)?

**Comment by devoncarew** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (yjbanov) [inline README.md]: done

File: bin/build.dart (hunk 2 above)
Comment (yjbanov) [inline bin/build.dart]: done

File: bin/build.dart (hunk 3 above)
Comment (yjbanov) [inline bin/build.dart]: cleaned everything in `build.dart` and `utils.dart`

File: lib/src/utils.dart (hunk 4 above)
Comment (yjbanov) [inline lib/src/utils.dart]: Done, here and some others.

**Pull request merged**